src/controllers/contactController.js

```diff
@@ -0,0 +1,18 @@
+const { sendMail } = require("./emailController");
```

Comment:
The email controller was an error from Rick. Please use the one in the services folder

Reply:
Whoops lol. I changed it and used HTML and everything still works for both files.